src/modules/snapshots/jobs/createSnapshotJob.ts

```diff
@@ -41,10 +43,15 @@ export async function createSnapshotJob(job: CreateSnapshotJob) {
   };
 
   await snapshotObjectRepository.upload({
-    environment: process.env.NODE_ENV === "production" ? "prod" : "local",
+    environment: getStorageEnvironment(),
     snapshot,
   });
   await snapshotRepository.create(snapshot);
+  await enqueueJob(SNAPSHOT_JOB_TYPES.CREATE_SNAPSHOT_INTERLINEAR_PDF, {
+    languageId: snapshot.languageId,
+    languageCode: language.code,
+    snapshotId: snapshot.id,
+  });
```

Comment:
The snapshot system is meant for backup/restore purposes, not to export for general consumption. We can remove PDF generation from the snapshot module